spec/fixtures/cassettes/features/spree/admin/checkout.yml

```diff
@@ -0,0 +1,116 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://api.taxjar.com/v2/taxes
+    body:
+      encoding: UTF-8
+      string: '{"to_country":"US","to_zip":"35242","to_city":"Richmond","to_state":"AL","to_street":"143
+        Swan Street","line_items":[{"id":1,"quantity":1,"unit_price":"19.99","discount":"-0.0","product_tax_code":"TaxCode
+        - 593989"}],"shipping":"10.0"}'
+    headers:
+      User-Agent:
+      - 'TaxJar/Ruby (Darwin Noahs-MacBook-Pro.local 20.6.0 Darwin Kernel Version
+        20.6.0: Mon Aug 30 06:12:21 PDT 2021; root:xnu-7195.141.6~3/RELEASE_X86_64
+        x86_64; ruby 2.6.7-p197; OpenSSL 1.1.1k  25 Mar 2021) taxjar-ruby/3.0.2'
+      Authorization:
+      - Bearer fake-bearer-token
```

Comment:
There is [a way to configure VCR to automatically filter out sensitive information](https://relishapp.com/vcr/vcr/v/5-0-0/docs/configuration/filter-sensitive-data).

I don't see us needing to re-record these request frequently, so maybe this would be overkill for us. But worth noting.

Reply:
It's a good thing to know about :). May be worth doing! It looks like the CI is failing, so will have to revisit this PR and might tackle that.

Reply:
Yes, we can definitely filter out certain headers and other parts of the request/response based on a regex https://relishapp.com/vcr/vcr/docs/configuration/filter-sensitive-data

Reply:
This SO seems to provide a good example for how to do this for the `Authorization` header https://stackoverflow.com/a/58629025/1492410